apps/editors/helpers.py

```diff
@@ -584,7 +584,7 @@ def __init__(self, request, addon, version, review_type):
         self.addon = addon
         self.version = version
         self.review_type = review_type
-        self.files = None
+        self.files = self.version.unreviewed_files if self.version else []
```

Comment:
For an addon review, we always review all the files. For a "files" review, we might want to only review one file. That's not possible using the forms (using the editor tools), but for example the `approve_addons` management command and the `auto_sign_file` helpers only sign a file at a time.